src/sphinx_book_theme/assets/scripts/index.js

```diff
@@ -230,3 +240,4 @@ sbRunWhenDOMLoaded(initTooltips);
 sbRunWhenDOMLoaded(scrollToActive);
 sbRunWhenDOMLoaded(initTocHide);
 sbRunWhenDOMLoaded(initRTDObserver);
+sbRunWhenDOMLoaded(addNoPrint);
```

Comment:
will this work if we print via headless chromium? That's what we use for HTML-to-PDF in jupyter book

Reply:
Hi @choldgraf , seems to be working. Primary sidebars and announcement banners etc. are hidden in pdfhtml build.